src/lib/database/pocky-db.ts

```diff
@@ -54,7 +55,11 @@ export default class PockyDB implements PockyDbInterface {
 			return dbConstants.pegError;
 		}
 
-		if (!senderHasPegs) {
+		// TODO: make this faster and less repetitive by refactoring getting the list of penaltyKeywords.
+		const penaltyKeywords = this.config.getStringConfig('penaltyKeyword');
+
+		if (!senderHasPegs
+			&&!penaltyKeywords.some(keyword => comment.toLowerCase().includes(keyword.toLowerCase()))) {
```

Comment:
Shouldn't this also check that there are no 'good' keywords in the message, since those don't count as shame pegs?

Reply:
Yes it should. Forgot to update this bit when switching from a default-penalty to default-good.